### DISCUSSION TITLE
The achievement doesn't appear in profile

### DISCUSSION THREAD

**Discussion by Alexponomarev7** [community/community#33583] [Other Feature Feedback, Questions, & Ideas]
My PR was merged https://github.com/ceph/ceph/pull/44321 in open-source project Ceph, but it doesn't appear in my profile. What should I do?

**Comment by dennyjon** [comment]
Select Game activity > Achievements, and then select the toggle switch to enable the Achievement tracker. Select the Settings icon, if you're not prompted automatically by the console, to determine where you'd like the tracker to be on your screen.

**Comment by Matteo-it** [comment]
Go on Settings > Public Profile and under Contributions & Activity check Include private contributions on my profile. Update preferences and you will see the badge!
